deepinfra/utils/read_stream.py

```diff
@@ -0,0 +1,67 @@
+import io
+import base64
+import requests
+
+class ReadStreamUtils:
+    """
+    Utility class for working with files.
+    """
+
+    @staticmethod
+    def buffer_to_stream(buffer):
+        """
+        Creates a BytesIO from a Buffer.
+        :param buffer: The Buffer to be streamed.
+        :return: A BinaryIO object.
+        """
+        return io.BytesIO(buffer)
+
+    @staticmethod
+    def file_to_stream(file_path):
+        """
+        Creates a BytesIO from a file path.
+        :param file_path: The path to the image file.
+        :return: A BinaryIO of the file contents.
+        """
+        return open(file_path, 'rb')
+
+    @staticmethod
+    def url_to_stream(url):
+        """
+        Downloads an image from a URL and returns it as a BytesIO.
+        :param url: The URL of the image.
+        :return: A BytesIO containing the image data.
+        """
+        response = requests.get(url)
+        return io.BytesIO(response.content)
+
+    @staticmethod
+    def base64_to_stream(base64_string):
+        """
+        Converts a Base64 string to a BytesIO.
+        :param base64_string: The Base64 string to be converted.
+        :return: A BytesIO of the image data.
+        """
+        image_data = base64.b64decode(base64_string)
+        return io.BytesIO(image_data)
+
+    @staticmethod
+    def get_read_stream(input_data):
```

Comment:
Python has the notion of a read stream (when you open a file with open you get that), so I think that should be supported as well.